lxqt-session/src/lxqtmodman.cpp

```diff
@@ -398,6 +398,15 @@ void LXQtModuleManager::logout(bool doExit)
         mWmProcess->kill();
     }
 
+    // Under wayland exit the compositor
+    if (QGuiApplication::platformName() == QLatin1String("wayland"))
+    {
+        QString program = QString::fromUtf8("exit_compositors");
```

Comment:
What's `exit_compositors`? If it's an external script, we can't its name in this code.

Reply:
Forgot to remove this, not necessary anymore.